repose-aggregator/components/filters/valkyrie-authorization-filter/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -34,6 +34,7 @@ import com.josephpconley.jsonpath.JSONPath
 import com.rackspace.httpdelegation.HttpDelegationManager
```

Comment:
It's actually the same parser, i'm just invoking it myself instead of through the other library.